go/vt/sqlparser/ast.go

```diff
@@ -1276,51 +1264,38 @@ func (node *ColName) Equal(c *ColName) bool {
 	return node.Name.Equal(c.Name) && node.Qualifier.Equal(c.Qualifier)
 }
 
+// ValTuple represents a list of value expressions.
+// It can be RowTuple or ParenExpr.
+type ValTuple interface {
+	iRowTuple()
+	Expr
+}
+
+func (RowTuple) iRowTuple()  {}
+func (ParenExpr) iRowTuple() {}
```

Comment:
Just tried it and deleting it doesn't compile. Definitely open to suggestions. I needed this to support the new if statement we added in sql.y -- the problem is:

row_tuple returned either a ParenExpr or a ValTuple, depending on the size of the value_expression_list. So I need a supertype that both of those fit into, to define as the return type of row_tuple.